beta/src/pages/learn/index.md

```diff
@@ -304,7 +304,7 @@ Often, you'll want your component to "remember" some information and display it.
 
 First, import [`useState`](/apis/usestate) from React:
 
-```js {1,4}
+```js
```

Comment:
?

Reply:
That code snippet is just 1 line but we have added highlight for 1 to 4 which is causing error in console